source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -100,26 +123,122 @@ The ovirt-provider-ovn provides an opinionated subset of OpenStack's networking
+adds some complexity during the network / subnet setup stage, but eases the
+configuration of all VMs attached to the networks whose subnets are connected
+to a router.
+
+Just creating the logical router is indeed not enough; all subnets that require
+RA messages to be multicasted into the connected VMs **must be** connected to
+at least one OVN logical router.
+
+That will create OVN logical flows that are responsible for generating the
+Router Advertisements messages the clients require to configure their IP
+addresses.
+These RA messages are generated as replies to the Router Solicitation messages.
+
+Periodic RA messages are also required, to prevent the default route on the
+guest VM from expiring; it has a default value of 65536 seconds.
+Simply activating the periodic flag on the logical router ports is enough, since
+its min/max defaults are well below that +-18 hour interval.
+
+### Routing - implementation detail
 
 The above change would be implemented in the following way:
+
 1. when a subnet is created, it's **ipv6_address_mode** parameter is stored in
 the subnet's external ids
 2. when that subnet is attached to a router, its **ipv6_ra_configs:address_mode**
 should be set from the subnet's external ids address mode value.
 
+## Guest VM configuration
+
+There are three different ways to configure the guests, enabling them to have
+IPv6 addresses, each with different advantages / drawbacks.
+
+### DHCPv6
+
+DHCPv6 is one option to configure the guests and requires **dhclient** to be
+installed in the guest VM.
+
+In dhcpv6_stateful, it is used to assign an IPv6 address to the specified
+interface, whereas in dhcpv6_stateless it is used to simply configure
+connection parameters - such as DNS.
+
+It is **not possible** to configure EL8 guests using only DHCPv6, since the
+dhclient version it ships relies on RA messages to configure the network prefix
+length. On EL8 guests, the guest will have to **also** accept RA messages, as
+indicated in the [next option](#linux-kernel-ra-configuration).
+
+### Linux Kernel RA configuration
+
+This option can be seen as a complement needed for el8 based VMs using dhclient,
+*or* a full fledged alternative to NetworkManager - with the advantage that it
+does not require any additional package to be installed in the guest VM.
+
+It involves configuring directy the behavior in the linux kernel, through the
+[interface config files](https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt).
+
+To follow it, the following files have to be created, and have a '1' in them:
+- accept_ra
+- autoconf
+- accept_redirects
+- accept_ra_pinfo
+
+And have the **forwarding** flag is disabled - write a '0' into the *forwarding*
+file
+
+All these files are located under the following folder:
+  - "/proc/sys/net/ipv6/conf/<YOUR_IF_NAME>/"
+
+### NetworkManager
+
+To use NetworkManager to configure the IPv6 address assignment, *nmcli* should
+be used. The recommended way can be found below:
+
```

Comment:
is this working for stateful and stateless?

Reply:
Looks like it.